lib/tasks/locale.rake

```diff
@@ -175,8 +175,8 @@ namespace :locale do
 
     pot_files = []
     Vmdb::Plugins.each do |plugin|
-      system('bundle', 'exec', 'rake', "locale:delete_pot_file[#{plugin.root}]") # Delete plugin's pot file if it exists to avoid weird file timestamp issues
-      raise unless system('bundle', 'exec', 'rake', "locale:plugin:find[#{plugin.to_s.sub('::Engine', '')}]")
+      Rake::Task['locale:delete_pot_file'].invoke(plugin.root) # Delete plugin's pot file if it exists to avoid weird file timestamp issues
+      Rake::Task['locale:plugin:find'].invoke(plugin.to_s.sub('::Engine', '')) # will warn and exit 1 if any engine fails
```

Comment:
Since this is run for each plugin, does it actually get run or does Rake "skip" it once it's been run at least once

Reply:
Ah good point... rake tasks won't be invoked a section time.  Let me check if different arguments change that.

Reply:
The time appears to stay the same... about 1 minute.  I'm currently checking if the contents are different vs. master.

Reply:
> Since this is run for each plugin, does it actually get run or does Rake "skip" it once it's been run at least once. Put another way, what do you see when you run it with `--trace`

Great find.  It's different now that I have all the plugins running and it looks like the global shared state is why this code was previously invoking a new shell session for each command.  Making this WIP.

Reply:
Ok, it took a while to test by generating on master and doing it over and generating on this branch to confirm that the files generated are the same.

This is the little hack that allows us to avoid creating new processes to run rake tasks in the context of each plugin:

```ruby
Rake.application.tasks.each {|t| t.reenable if (t.already_invoked && !t.kind_of?(Rake::FileTask))}
```

Reply:
The solution should be to change some of these rake tasks to method so we can call the method multiples times in the same rake tasks and not skip it.  Additionally, we can avoid loading environment if we have to go back to using extra process invocations but I think this solution is simple enough until we can fix the underlying rake tasks so they can cleanup after themselves and not be skipped when run for each plugin.